app/assets/javascripts/utilities/buildArticleHTML.js

```diff
@@ -2,7 +2,7 @@
 
 /* eslint-disable no-multi-str */
 
-function buildArticleHTML(article) {
+function buildArticleHTML(article, currentUserId = null) {
```

Comment:
`buildArticleHTML` isn't called in many places, but I still felt defaulting `currentUserId` to `null` was good practice 😅

Reply:
TBH it won't make much difference in JS land, as if `currentUserId` isn't passed in, it will just have value `undefined` and your other equality checks would handle it as-is (fine either way though)